values.yaml

```diff
@@ -182,55 +103,14 @@ envproxy:
   # disconnects during deployments.
   terminationGracePeriodSeconds: 14400
 
-# timescale -- Contains configuration for the internal database. It is not
-# recommended to run this service in production. See the `postgres` section for
-# connecting to an external Postgres database.
-timescale:
-  # timescale.image -- Injected during releases.
-  image: ""
-  # timescale.resources -- Kubernetes resource request and limits for the
-  # timescale pod.
-  # To unset a value, set it to "".
-  # To unset all values, you can provide a values.yaml file which sets resources
-  # to nil. See values.yaml for an example.
-  #
-  # e.g:
-  # timescale:
-  #   # This will cause all values to be unset.
-  #   resources:
-  resources:
-    requests:
-      cpu: "250m"
-      memory: "1Gi"
-      # timescale.resources.requests.storage -- Specifies the size of the
-      # volume claim for persisting the database.
-      storage: "10Gi"
-    limits:
-      cpu: "250m"
-      memory: "1Gi"
-
-envbox:
-  # envbox.image -- Injected during releases.
-  image: ""
-
-envbuilder:
-  # envbuilder.image -- Injected during releases.
-  image: ""
-
-# environments defines configuration that is applied to all
-# user environments. 
-environments:
-  # environments.tolerations -- Tolerations are applied to all user
-  # environments. Each element is a regular pod toleration object. To set
-  # service tolerations see serviceTolerations. See values.yaml for an example.
-  #
-  # e.g.
-  # tolerations:
-  # - key: "my key"
-  #   operator: "Exists"
-  #   effect: "NoExecute"
-  #   tolerationSeconds: 6000
-  tolerations: []
+# cemanager -- Details for connecting to the cemanager instance.
+# NOTE: the envproxy must be accessible by envproxy.accessURL from the
+# cemanager.
```

Comment:
should this note be on the envproxy accessURL instead? And here we say "cemanager must be accessible"